.github/steps/setup-repo/action.yml

```diff
@@ -16,3 +16,9 @@ runs:
       uses: actions/setup-node@v4
       with:
         registry-url: "https://npm.pkg.github.com"
+
+    - name: Disable npm scripts
```

Comment:
Looks good to me, but after merging, can you verify that we can indeed not execute scripts?